src/components/info/byline.js

```diff
@@ -106,11 +109,13 @@ function renderMaintainers(t, metadata) {
  * Returns a React component detailing the date the data was last updated.
  * Renders a <span> containing "Data updated X", where X derives from `metadata.updated`
  */
-function renderDataUpdated(t, metadata) {
-  if (metadata.updated) {
+function renderDataUpdated(t, metadata, language) {
+  if (moment(metadata.updated).isValid()) {
     return (
       <span>
-        {`${t("Data updated")} ${metadata.updated}. `}
+        {`${t("Data updated")} `}
+        <Moment locale={language} date={metadata.updated} fromNow />
```

Comment:
(I haven't run the code, just reading it)

How does this work for very recent dates? In `augur export` the updated string uses [`time.strftime('%Y-%m-%d')`](https://github.com/nextstrain/augur/blob/master/augur/export_v2.py#L1086) which'll use the [users local time](https://docs.python.org/3/library/time.html#time.strftime). Our schema [says nothing](https://github.com/nextstrain/augur/blob/master/augur/data/schema-export-v2.json#L24C1-L28C19) about time zones. I presume (?) moment's also interpreting the date string in the (Auspice) users' local time zone (or is it UTC?). 

So what happens if, e.g., I update a dataset right now, thus getting "2023-12-13" and someone in the US views it (the US is currently Dec 12th)... is the dataset updated "tomorrow"? Do we ignore it because it's a bad value? What about the inverse situation, would the dataset appear to me to be updated "yesterday" even if it'd been updated 5 min ago? Time zones make this hard!

Reply:
It'd be useful to create a table or similar testing different values here. From the datasets on the Auspice review app, long-ago updated datasets do look better than before, e.g. "Data updated 2 years ago" is better than a YYYY-MM-DD string!

Reply:
Good point, and very important to consider since we have daily ncov builds.

There's really no good way to handle this with the current schema. The same thing could be said for the current implementation of displaying `meta.updated` as-is without any time zone info.

I think the difference is that relative dates must assume a time zone, and that its effects are much worse when recent. Continuing your example, "tomorrow" sounds much worse than "2023-12-13", but "3 years ago" zooms the perspective out to a point where time zone doesn't matter.

How about we only show relative dates if it's at least a week old?

Reply:
Going further, we could extend the schema to support an exact ISO datetime (e.g. `2023-12-12T21:47:18Z`) which then allows proper conversion to the user's timezone.